src/sharepoint/order/sharepoint-order.html

```diff
@@ -1,125 +1,90 @@
-                                        <label class="d-block font-weight-normal m-0 p-3" for="account.primaryEmailAddress.{{$index}}"
-                                               data-ng-bind="account.primaryEmailAddress"></label>
-                                    </td>
-                                </tr>
-                            </tbody>
-                        </table>
+                        <oui-field data-label="{{ 'sharepoint_activate_which_service_label' | translate }}">
+                            <oui-select data-id="sharepointAssociatedExchange"
+                                        data-name="sharepointAssociatedExchange"
+                                        data-on-change="$ctrl.refreshAccounts(modelValue)"
+                                        data-disabled="$ctrl.isComingFromAssociatedExchange"
+                                        data-model="$ctrl.associatedExchange"
+                                        data-items="$ctrl.exchanges"
+                                        data-match="displayName"
+                                        data-required>
+                                <span ng-bind="$item.displayName"></span>
+                            </oui-select>
+                        </oui-field>
 
-                        <div class="clearfix"
-                             data-ng-if="orderCtrl.accountsIds">
-                            <div data-pagination-front
-                                 data-items="orderCtrl.accountsIds"
-                                 data-paginated-items="orderCtrl.accountsList"
-                                 data-current-page="currentPage"
-                                 data-items-per-page="itemsPerPage"
-                                 data-nb-pages="nbPages"
-                                 data-transform-item="orderCtrl.onTranformItem(item)"
-                                 data-on-transform-item-done="orderCtrl.onTranformItemDone()"
-                                 data-page-placeholder="{{ 'pagination_page' | translate: { current: currentPage, last: nbPages } }}"
-                                 data-item-per-page-placeholder="{{ 'pagination_display' | translate }}">
-                            </div>
-                        </div>
+                        <oui-datagrid id="exchangeAccountsDatagrid"
+                                      data-rows-loader="$ctrl.getAccounts()"
+                                      data-row-loader="$ctrl.getAccount($row)"
+                                      data-on-row-select="$ctrl.onAccountsSelected($rows)"
+                                      data-selectable-rows>
+                            <oui-column data-title="'sharepoint_accounts_userPrincipalName' | translate"
+                                        data-property="primaryEmailAddress"
+                                        data-empty-placeholder="{{ 'sharepoint_accounts_empty' | translate }}">
```

Comment:
```suggestion
                                         data-empty-placeholder="{{:: 'sharepoint_accounts_empty' | translate }}">
```
Also the `empty-placeholder` should be on `datagrid` and not on `column`